web/modules/custom/collection/collection.module

```diff
@@ -563,8 +564,14 @@ function collection_search_api_solr_query_alter(Query $solarium_query, QueryInte
  * Implements hook_ENTITY_TYPE_presave().
  */
 function collection_og_membership_presave(OgMembershipInterface $entity) {
-  $tags = [$entity->getGroupEntityType() . ':' . $entity->getGroupId()];
-  Cache::invalidateTags(Cache::buildTags('joinup-membership', $tags));
+  joinup_core_og_membership_invalidate_group_tags($entity);
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_predelete().
+ */
+function collection_og_membership_predelete(OgMembershipInterface $entity) {
```

Comment:
Since these functions are about og_membership in general, should we move it together with the `joinup_core_user_update()` function that you just wrote? I didn't think about this before, your new code exposed it! 😝

Reply:
I was thinking of it as well and I was struggling a bit. The reason it is in collections is that it is collections that have the leave button and the members count. However, I think that it is a good option as well to keep everything cache related gathered in joinup_core.